gradle.properties

```diff
@@ -16,8 +16,10 @@
 projectGroup=com.ritense.valtimo
 projectName=valtimo-backend-libraries
 kotlinVersion=1.6.10
-springBootVersion=2.5.12
-springDependencyManagementVersion=1.0.11.RELEASE
+springBootVersion=2.7.4
+springBootAdminStarterClientVersion=2.7.5
+springDependencyManagementVersion=1.0.14.RELEASE
```

Comment:
I hate so say it, but 1.1.0 has been released today.

Reply:
Wow, nice timing!